src/main/java/io/codelex/classesandobjects/practice/videostore/VideoStoreTest.java

```diff
@@ -29,28 +39,80 @@ public static void main(String[] args) {
                 case 3:
                     returnVideo(keyboard);
                     break;
+                case 4:
+                    printInStockInventory();
+                    break;
                 default:
                     break;
             }
-
         }
     }
 
     private static void fillVideoStore(Scanner scanner) {
-        for (int i = 0; i < COUNT_OF_MOVIES; i++) {
-            System.out.println("Enter movie name");
-            String movieName = scanner.next();
-            System.out.println("Enter rating");
+        for (int i = 0; i < 1; i++) {
+            System.out.println("Enter movie name ");
+            String movieName = scanner.nextLine();
+            System.out.println("Enter rating ");
             int rating = scanner.nextInt();
-            //todo - add video
+            scanner.nextLine(); // after nextInt clean terminal
+            VideoStore.addNewVideo(movieName, rating);
         }
     }
 
     private static void rentVideo(Scanner scanner) {
-        //todo - rent video
+        System.out.println("Enter movie name ");
+        String movieName = scanner.nextLine().toLowerCase();
+
+        for (Video video : VideoStore.inStock) {
+            if (video.checkOut(movieName)) {
+                if (video.getAvailable().equals(false)) {
+                    System.out.println("Already rented. Please choose another.");
+                    printInStockInventory();
+                    System.out.println();
+                    menu();
+
+                } else if (video.getAvailable().equals(true)) {
```

Comment:
nav nepeiciešams `equals`, var nedaudz parsaukt metodi un tiaki to lietot iekš if `else if(video.isAvailable()`

Reply:
Paldies, izlaboju.